android/package.json

```diff
@@ -40,6 +40,6 @@
     "flow-bin": "0.63.1",
     "haul": "1.0.0-rc.4",
     "prettier": "1.8.2",
-    "webpack": "4.12.0"
+    "webpack": "4.6.0"
```

Comment:
how come reverting this to 4.12?

Reply:
I updated this to 4.12.0 while testing android issues, but times-components are still on 4.6.0. I rather keep it consistent for now